pom.xml

```diff
@@ -457,7 +457,7 @@
             <dependency>
                 <groupId>com.clickhouse</groupId>
                 <artifactId>clickhouse-jdbc</artifactId>
-                <version>0.6.0-patch4</version>
+                <version>0.6.0-patch5</version>
```

Comment:
This version and `partch4` are still pre-release. I think we shouldn't use it. 
https://github.com/ClickHouse/clickhouse-java/releases/tag/v0.6.0-patch5

Reply:
As far as I understand the project structure these are patch releases to 0.6.0 release.

Reply:
I know they are patch releases. However, `patch3` is the latest release on GitHub. Is it really safe to use 4 and 5? 
![Screenshot 2024-05-28 at 7 20 17](https://github.com/trinodb/trino/assets/6237050/cb566a06-7ab0-475d-b564-8bdfaf8ba0be)